Tools-Override/FrameworkTargeting.targets

```diff
@@ -126,28 +126,22 @@
     <!-- list each append as a seperate item to force re-evaluation of AdditionalProperties metadata -->
     <ItemGroup>
       <ProjectReference>
-        <UndefineProperties Condition="'$(ConfigurationGroup)' != ''">%(ProjectReference.UndefineProperties);Configuration</UndefineProperties>
-      </ProjectReference>
-
-      <!-- Set some basic configuration properties and pass them along to ProjectReference's -->
-      <ProjectReference>
-        <OSGroup Condition="'%(ProjectReference.OSGroup)'=='' and '$(OSGroup)'!='' and '$(OSGroup)'!='AnyOS'">$(OSGroup)</OSGroup>
-      </ProjectReference>
-      <ProjectReference>
-        <TargetGroup Condition="'%(ProjectReference.TargetGroup)'=='' and '$(TargetGroup)'!=''">$(TargetGroup)</TargetGroup>
-      </ProjectReference>
-
-      <!-- Configuration property shortcuts -->
-      <ProjectReference>
-        <AdditionalProperties Condition="'%(ProjectReference.TargetGroup)'!=''">TargetGroup=%(ProjectReference.TargetGroup);%(ProjectReference.AdditionalProperties)</AdditionalProperties>
-      </ProjectReference>
-      <ProjectReference>
-        <AdditionalProperties Condition="'%(ProjectReference.OSGroup)'!=''">OSGroup=%(ProjectReference.OSGroup);%(ProjectReference.AdditionalProperties)</AdditionalProperties>
-      </ProjectReference>
-      <ProjectReference>
-        <AdditionalProperties Condition="'$(ConfigurationGroup)' != ''">ConfigurationGroup=$(ConfigurationGroup);%(ProjectReference.AdditionalProperties)</AdditionalProperties>
+<!-->        <UndefineProperties Condition="'$(ConfigurationGroup)' != ''">%(ProjectReference.UndefineProperties);Configuration</UndefineProperties> -->
       </ProjectReference>
 
+     <!-- Configuration property shortcuts --> 
```

Comment:
I'm not convinced that we even need to keep these shortcuts any longer. @ericstj do you think they are interesting for any scenarios now?